x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/services/analytics_service/get_analytics.ts

```diff
@@ -39,20 +42,70 @@ const isGetDataFrameAnalyticsStatsResponseOk = (
   );
 };
 
-interface GetDataFrameAnalyticsStatsResponseError {
-  statusCode: number;
-  error: string;
-  message: string;
-}
+export type GetAnalytics = (forceRefresh?: boolean) => void;
 
-type GetDataFrameAnalyticsStatsResponse =
-  | GetDataFrameAnalyticsStatsResponseOk
-  | GetDataFrameAnalyticsStatsResponseError;
+/**
+ * Gets initial object for analytics stats.
+ */
+export function getInitialAnalyticsStats(): AnalyticStatsBarStats {
+  return {
+    total: {
+      label: i18n.translate('xpack.ml.overview.statsBar.totalAnalyticsLabel', {
+        defaultMessage: 'Total analytics jobs',
+      }),
+      value: 0,
+      show: true,
+    },
+    started: {
+      label: i18n.translate('xpack.ml.overview.statsBar.runningAnalyticsLabel', {
+        defaultMessage: 'Running',
+      }),
+      value: 0,
+      show: true,
+    },
+    stopped: {
+      label: i18n.translate('xpack.ml.overview.statsBar.stoppedAnalyticsLabel', {
+        defaultMessage: 'Stopped',
+      }),
+      value: 0,
+      show: true,
+    },
+    failed: {
+      label: i18n.translate('xpack.ml.overview.statsBar.failedAnalyticsLabel', {
+        defaultMessage: 'Failed',
```

Comment:
Are failed jobs showing up correctly in the stats bar? I have 2 failed jobs, but I can't see them in any of the stats bars:

![image](https://user-images.githubusercontent.com/7405507/68590471-77303c00-0486-11ea-925a-0e33e918fb40.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/49464/commits/9a1aa0a6e8c74577832f4a3c1699b95164917fad